examples/t5/schedule.py

```diff
@@ -141,7 +141,7 @@ def reshape_for_scores(self, x):
                 def forward(self, hidden_states):
                     states = self.query(hidden_states)
                     states = self.reshape_for_scores(states)
-                    return [states]
+                    return states
```

Comment:
Is this a bug I did before?

Reply:
It is actually a bug of `replace_module`. See these several lines of code:
https://github.com/awslabs/slapo/blob/3393bdfbf8c60370736906381c6111cfc16369a0/slapo/schedule.py#L770-L772
Currently `subgraphs` is a list of list of tuple, which means the if-condition is always true. After fixing the condition, `pattern_q` should go through the vertical fusion branch, and thus the return of the replaced module is just a tensor instead of a list.